src/index.html

```diff
@@ -11,7 +11,145 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>HTML Form</h1>
     <script type="text/javascript" src="./main.js"></script>
+    <form action="https://mate-academy-form-lesson.herokuapp.com/create-application" method="post">
+      <fieldset class="fields">
+        <legend>
+          Personal information
+        </legend>
+        <label class="first-field">
+          Surname:
+          <input
+            type="text"
+            name="surname"
+            autocomplete="off"
+            required>
+        </label>
+
+        <label class="form-field">
+          Name:
+          <input
+            type="text"
+            name="name"
+            autocomplete="off"
+            required>
```

Comment:
```suggestion
          <input
            type="text"
            name="name"
            autocomplete="off"
            required
          >
```
закрывающий тег должен быть на новой строке